web_src/js/components/RepoContributors.vue

```diff
@@ -389,7 +397,11 @@ export default {
               {{ contributor.name }}
             </h4>
             <p class="tw-text-12 tw-flex tw-gap-1">
-              <strong v-if="contributor.total_commits">{{ contributor.total_commits.toLocaleString() }} {{ locale.contributionType.commits }}</strong>
+              <strong v-if="contributor.total_commits">
+                <a class="silenced" :href="searchQuery + contributor.email">
```

Comment:
It needs to encode (escape) the query parameter correctly

I think you need to use "computed" value instead of using `+` here.

Reply:
resolved in 5d9bf83667c08509f12f42c17f74154d2b631f7c
Added a method to compute the search query for each contributor